Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1393,6 +1331,18 @@ namespace OpenInfraPlatform {
 				}
 			}
 
+			void SearchExistingVertex(std::stringstream ID, uint32_t index, carve::geom::vector<3> position) const

```

Comment:
`index` needs to be passed per reference `&`, otherwise the value will not be available outside the function (within the calling scope).

Reply:
Resolved: 1b6fe62